algebird-test/src/test/scala/com/twitter/algebird/CountMinSketchTest.scala

```diff
@@ -930,7 +930,7 @@ class CMSFunctionsSpec extends PropSpec with PropertyChecks with Matchers {
   }
 
   property("throw IAE when deriving depth from invalid delta values") {
-    val invalidDeltas = Table("invalidDelta", 0.0, 1E-330, 1E-400)
```

Comment:
these numbers were invalid doubles and 2.12.3 won't parse them. scala 2.11 I think parses them to 0.0.